### PR TITLE
home page desription

### LINKED ISSUE #6: Storybook: Description Career Advice Option component
## User story

**Who:** **As a** user
**What:** **I want to** be able to see the summary for the Career Advices options
**Why:** **so that I can** choose which one to explore

## Acceptance criteria

Create ONE(1) storybook component for the Description Career Advice Option like the ones we can see highlighted in pink 
 
Screenshot 2021-04-13 at 00 33 02

Check the wireframes here: https://www.figma.com/file/9SJRaxPSuPI3lSl8ULNyJ8/GuideIT-For-Developers?node-id=0%3A1

## Implementation details

Use the font [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono?preview.text=What%20is%20GuideIT%3F&preview.text_type=custom) and colors #DB643D and #535C75 for text and color #E9F4F4 for the background

### PR DESCRIPTION
**Pull request by abedaarabi**
# Description

add home page description

Fixes #6 

# How to test?
switch to description/career branch 

run: npm run storybook

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by dpfernandes — CHANGES_REQUESTED** [review]
Please don't add env.example to the project. 

Can you remove it please? :)

**Review by soujanyapolavarapu — APPROVED** [review]
I cloned and checked ,It look no errors.

**Comment by abedaarabi** [comment]
> Please don't add env.example to the project.
> 
> Can you remove it please? :)

I have done that :)

File: src/client/components/Description Career/descriptioncareer.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400&display=swap');
+
+.descriptioncareer {
+  padding: 10px;
+  padding-left: 20px;
+  background-color: #e9f4f4;
+  margin: 10px;
```
Comment (TITHI1244) [inline src/client/components/Description Career/descriptioncareer.css:7]: I think, you can use a fixed height (I tried with 95%) for this class, the container to make them look equally long as design :)

File: src/client/components/Description Career/descriptioncareer.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  display: grid;
+  grid-template-columns: 50% 50%;
+  justify-content: space-between;
+}
+
+.descriptioncareer .title {
+  color: #db643d;
+  font-family: 'Roboto Mono', monospace;
+  font-style: normal;
+  font-size: 28px;
```
Comment (TITHI1244) [inline src/client/components/Description Career/descriptioncareer.css:20]: One more thing, if you change the font-size into 25px, it looks more identical with the actual design I think :)

**Review by TITHI1244 — APPROVED** [review]
Good job @abedaarabi. Nicely done :)

File: src/client/components/Description Career/descriptioncareer.css (hunk 2 above)
Comment (abedaarabi) [reply src/client/components/Description Career/descriptioncareer.css:20]: thank you @TITHI1244  will do that 👍

**Comment by dpfernandes** [comment]
This is being handled here instead https://github.com/HackYourFuture-CPH/Guide-IT/pull/71

**Pull request closed without merging**